packages/ng/button/button.component.ts

```diff
@@ -15,6 +16,7 @@ import { NgClazz, Palette } from '@lucca-front/ng/core';
 })
 export class ButtonComponent implements OnChanges {
 	#ngClazz = inject(NgClazz);
+	#elementRef = inject(ElementRef);
```

Comment:
To avoid `as HTMLButtonElement` later, you can type it here

```suggestion
	#elementRef = inject<ElementRef<HTMLButtonElement>>(ElementRef);
```